docs/Actions/inject-data.md

```diff
@@ -0,0 +1,273 @@
+| Name/Value | First parameter refers to the name of the variable, the second one refers to the value assigned. The action use a key-value(name-value) concept. <br></br> Each data that you want to inject has different parameters. | Yes |
+
+## `Output Parameters Reference`
+
+| Parameter | Description |
+|-----------|-------------|
+| Output Token Name | The Name parameter is stored as a token for further use anywhere in the form. |
+
+## `Examples`
+
+### `1. Inject data using Run SQL Query`
+
+​
+The action below selects first client name from a database using Run SQL Query and sends it into the form. <br></br>[Import it](/docs/Actions/import-actions.md) into your application to see it in action.
+​
+
+```json
+{
+    "Title": "Execute Actions",
+    "ActionType": "ExecuteActions",
+    "Description": "Extract and inject first client name into form",
+    "Condition": null,
+    "Parameters": {
+        "ActionList": [
+            {
+                "Id": -1,
+                "$_uid": "action15911016148437876",
+                "Parameters": {
+                    "ConnectionString": "",
+                    "QueryTimeout": "",
+                    "SqlQuery": "SELECT Name FROM app.Client\nWHERE Id = 1",
+                    "BindTokens": [],
+                    "OutputTokenName": "",
+                    "ExtractColumns": [
+                        {
+                            "value": "SQL_Name",
+                            "name": "Name"
+                        }
+                    ],
+                    "OnError": [],
+                    "ShowErrors": false
+                },
+                "ActionType": "RunSql",
+                "$_isOpen": false,
+                "$_isLoaded": true,
+                "$_isFocus": true,
+                "Definition": {
+                    "IsClientAction": false,
+                    "Settings": {
+                        "Group": "Data"
+                    }
+                }
+            },
+            {
+                "Id": -1,
+                "$_uid": "action15911016148431102",
+                "Parameters": {
+                    "Data": [
+                        {
+                            "value": "[SQL_Name]",
+                            "name": "Inject_Name"
+                        }
+                    ]
+                },
+                "ActionType": "InjectData",
+                "$_isOpen": false,
+                "$_isLoaded": true,
+                "$_isFocus": true,
+                "Definition": {
+                    "IsClientAction": false,
+                    "Settings": {
+                        "Group": "Context"
+                    }
+                }
+            }
+        ]
+    }
+}
+```
+
+### `2. Generate a new token`
+
+​
+The action below generates a new token named "Client_ID" with the assigned value "ID15367" that can be used in the form. <br></br> [Import it](/docs/Actions/import-actions.md) into your application to see it in action.
```

Comment:
Let's prefer creating variables like ClientId instead of Client_ID. It looks much more modern.

Reply:
Fixed it